### PR TITLE
False positives, fixed by nestedQuery 

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search/resource/ResourceStreamBuilders.java [hunk 1]
```diff
@@ -125,70 +132,28 @@ public Stream<Map.Entry<ResourceParameter, QueryBuilder>> additionalIdentifierQu
...
-                                        jsonPath(
-                                                ENTITY_DESCRIPTION,
-                                                CONTRIBUTORS,
-                                                IDENTITY,
-                                                VERIFICATION_STATUS,
-                                                KEYWORD),
-                                        NOT_VERIFIED))
-                        .minimumShouldMatch(1);
+                nestedQuery(
+                        ENTITY_CONTRIBUTORS,
```
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search/resource/ResourceStreamBuilders.java:136]: The actual code change....

File: search-commons/src/main/java/no/unit/nva/search/resource/Constants.java
Comment (StigNorland) [inline search-commons/src/main/java/no/unit/nva/search/resource/Constants.java:1]: Applied jsonPath to all constants

File: search-commons/src/test/resources/resource_urls.json [hunk 2]
```diff
@@ -214,7 +214,7 @@
   "TOP_LEVEL_ORGANIZATION=194.31.5.0": 0,
   "TOP_LEVEL_ORGANIZATION=194.99.9.9": 0,
   "TYPE_should=ReportResearch,AcademicArticle": 17,
-  "UNIDENTIFIED_NORWEGIAN=true": 4,
+  "UNIDENTIFIED_NORWEGIAN=true": 2,
```
Comment (StigNorland) [inline search-commons/src/test/resources/resource_urls.json:217]: eqaul no false positives

File: search-commons/src/main/java/no/unit/nva/search/resource/Constants.java [hunk 3]
```diff
@@ -100,29 +98,28 @@ public final class Constants {
...
-    public static final String ENTITY_PUBLICATION_INSTANCE_DOT =
-            ENTITY_DESCRIPTION + DOT + REFERENCE + DOT + PUBLICATION_INSTANCE + DOT;
+    public static final String ENTITY_PUBLICATION_INSTANCE =
+            jsonPath(ENTITY_DESCRIPTION, REFERENCE, PUBLICATION_INSTANCE);
 
     public static final String CONTRIBUTOR_ORG_KEYWORD =
             jsonPath(CONTRIBUTOR_ORGANIZATIONS, KEYWORD);
 
     public static final String CONTRIBUTORS_AFFILIATION_ID_KEYWORD =
-            ENTITY_CONTRIBUTORS_DOT + AFFILIATIONS + DOT + ID + DOT + KEYWORD;
```
Comment (truhacevkir) [inline search-commons/src/main/java/no/unit/nva/search/resource/Constants.java:122]: nice

**Review by truhacevkir — APPROVED** [review]

**Pull request merged**